external/fv3fit/fv3fit/train_microphysics.py

```diff
@@ -204,6 +208,93 @@ def __post_init__(self) -> None:
             self.cache = False
 
 
+def get_model_output_sensitivities(
+    model: tf.keras.Model, sample: Mapping[str, tf.Tensor],
+):
```

Comment:
Output type hint please. mypy does nothing if there is no output type and it is nice to know at a glance what this function returns since the docstring isn't clear and the code is long.

This function does a lot...I'd consider separating the "jacobian of a dict" part from the "compute mean and normalize output part".

Reply:
Separated the jacobian  calculation from the jacobian normalizing.   Added the specifics for the functions to `emulation/keras.py` and the plotting to `wandb.py`.